elegant/clean_timepoint_data.py

```diff
@@ -113,11 +122,30 @@ def remove_excluded_positions(experiment_root,dry_run=False):
     annotations = load_data.read_annotations(experiment_root)
     good_annotations = load_data.filter_annotations(annotations, load_data.filter_excluded)
     excluded_positions = sorted(set(annotations.keys()).difference(set(good_annotations.keys())))
+
     for position in excluded_positions:
         if (experiment_root / position).exists():
             print(f'Found an excluded position to delete {position}')
             if not dry_run:
+                (experiment_root / 'excluded_positions' / position).mkdir(parents=True,exist_ok=True)
+                (experiment_root / 'excluded_positions' / 'annotations').mkdir(parents=True,exist_ok=True)
+
+                shutil.copy(str(experiment_root / position / 'position_metadata.json'),
+                    str(experiment_root / 'excluded_positions' / position / 'position_metadata.json'))
+                shutil.copy(str(experiment_root / 'annotations' / f'{position}.pickle'),
+                    str(experiment_root / 'excluded_positions' / 'annotations' / f'{position}.pickle'))
+                shutil.copy(str(experiment_root /  'experiment_metadata.json'),
+                    str(experiment_root / 'excluded_positions' / 'experiment_metadata_old.json')) # Back this up JIC....
```

Comment:
why call it `_old` when everything else gets its own filename?

One possibility: if it already exists in the excluded root, try incrementing a counter so you get a save of every version of the file. But that is probably overkill.